sematic/resolvers/local_resolver.py

```diff
@@ -96,6 +96,8 @@ def _seed_from_clone(self, future: AbstractFuture, from_run_id: str):
 
         self._futures = list(cloned_graph.futures_by_original_id.values())
 
+        self._runs.clear()
```

Comment:
This is necessary, otherwise the root run will not be updated with its cloned status.

In detached execution, or rerun, the root run is created outside the resolver (either by the server, or by the resolver submitting the detached resolution from the user's machine).

Reply:
The alternative would be either
- some nasty logic in `populate_graph`
- not calling `set_graph` in `worker.py`
I chose the option implemented here because it is an idiosyncrasy of the re-run behavior and this confines the change to `_seed_from_clone` which is only called for re-runs.

Reply:
Comment added.